content/blog/hackathon-2022-wazo-matrix-bridge.md

```diff
@@ -0,0 +1,60 @@
+Title: Hackathon 2022 - Bridging Wazo and Matrix
+Date: 2022-10-20 12:00:00
+Author: Jesse Sopel
+Category: Hackathon
+Tags: Chat, Python, Matrix, Bridge, Hackathon
+Slug: hackathon-2022-bridging-wazo-and-matrix
+Status: published
+
+# Hackathon 2022 - Bridging Wazo and Matrix
```

Comment:
No need to add the main title in the content, the website will that the title in meta-data (aka line #1)

Reply:
Oh thanks. I wasn't sure how it was rendered.

Reply:
FYI: I did the change since I reformated the header with the new syntax.

Reply:
Oh perfect, thanks!